internal/stolonctl/pkg/cluster/config.go

```diff
@@ -51,3 +62,12 @@ func (c *Config) CheckConfig() error {
 	}
 	return nil
 }
+
+// CheckPostgresParams checks provided configuration for PostgreSQL connection
+func (c *Config) CheckPostgresParams() error {
+	if c.PostgresHost == "" || c.PostgresPort == "" ||
+		c.PostgresUser == "" || c.PostgresBackupPath == "" {
+		return trace.BadParameter("postgres-host, postgres-port, postgres-user and postgres-backup-path are required")
```

Comment:
We usually validate  each parameter individually - this makes for better error messages. You can collect errors and return multiple with `trace.NewAggregate`